src/topojson/topojson.ml

```diff
@@ -479,13 +482,22 @@ module Make (J : Intf.Json) = struct
           List.filter (fun (k, _v) -> not (List.mem k keys_in_use)) assoc
       | Error _ -> []
 
-    let of_json json =
+    let transform_of_json json =
```

Comment:
The implementation here doesn't look quite right. I think we will need to use `J.find` to get at the transform object. As well, when decoding the scale and translate values we can convert them to arrays (that's what they are in JSON because JSON doesn't have the notion of pairs). Something like 


```ocaml
let transform_of_json json =
      match J.find json [ "transform "] with
      | None -> None
      | Some transform_object ->
        let scale = decode_or_err (J.to_array (decode_or_err J.to_float)) transform_object in
        let translate = decode_or_err (J.to_array (decode_or_err J.to_float)) transform_object in
        Some { scale = scale.(0), scale.(1); translate = translate.(0), translate.(1) }
```

Does that make sense, note as well how it now returns an optional value. Previously you had `{}` which isn't valid OCaml, but I think I get what you wanted to do, in OCaml we can represent that "nothingness" with `None`.